rest_api/sawtooth_rest_api/messaging.py

```diff
@@ -161,10 +182,20 @@ def __init__(self, socket, msg_router):
             content=message_content,
             message_type=message_type)
 
-        try:
-            await self._socket.send_multipart([message.SerializeToString()])
-        except asyncio.CancelledError:
-            raise
+        # Send the message. Backoff and retry in case of an error
+        backoff = _Backoff(max_retries=3,
```

Comment:
Do we have a reason we would only retry 3 times?  Is there a total time we are trying to stay under?

Similarly, why was "interval=200" chosen? If there is reasoning behind these numbers, it could be added to the comment above this statement.